src/OpenSage.Game/Gui/Apt/RenderItem.cs

```diff
@@ -45,7 +50,8 @@ public override void Render(AptRenderer renderer, ItemTransform pTransform, Draw
                     }
 
                     //localize our content
-                    t.Content = t.Content.Replace("$", "APT:");
+                    t.Content = t.Content.Replace("$", "APT:"); // All string values begin with $
+                    t.Content = t.Content.Split('&').First();   // Query strings after ampersand
```

Comment:
There was stuff like `APT:PlayerRank&dropShadow`. The actual string was without the &dropShadow though